### PR TITLE
Update recommended VS Code extensions

### PR DESCRIPTION
**Pull request by tomheaton**
Closes #(issue)

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
 		"tauri-apps.tauri-vscode",
 		"rust-lang.rust-analyzer",
 		"oscartbeaumont.rspc-vscode",
-		"EditorConfig.EditorConfig",
-		"bradlc.vscode-tailwindcss"
+		"editorconfig.editorconfig",
+		"bradlc.vscode-tailwindcss",
+		"prisma.prisma"
```
Comment (brxken128) [inline .vscode/extensions.json:8]: This should be `Prisma.prisma`.

We could also add `esbenp.prettier-vscode` and `dbaeumer.vscode-eslint` I reckon, as they are both rather helpful and I'm pretty sure most of the team use them.

Comment (tomheaton) [reply .vscode/extensions.json:8]: > This should be `Prisma.prisma`.
fun fact is it is not case sensitive

Comment (tomheaton) [reply .vscode/extensions.json]: have added the others

Comment (brxken128) [reply .vscode/extensions.json]: > > This should be `Prisma.prisma`.
> 
> fun fact is it is not case sensitive
> 
> 

well you changed the case of the editorconfig extension so i was just sticking with what changes you previously made

**Review by brxken128 — APPROVED** [review]

**Pull request merged**